overlaycheck/CMakeLists.txt

```diff
@@ -1,8 +1,10 @@
 cmake_minimum_required(VERSION 3.10)
 
+include(GNUInstallDirs)
+
 #set project name
 project(overlaycheck)
 
 #add executables
-install(PROGRAMS overlaycheck DESTINATION bin)
-install(FILES overlaycheck_exclusions.txt DESTINATION bin)
+install(PROGRAMS overlaycheck DESTINATION ${CMAKE_INSTALL_BINDIR})
+install(FILES overlaycheck_exclusions.txt DESTINATION ${CMAKE_INSTALL_BINDIR})
```

Comment:
Architecture-independent data shouldn't really be shipped in /usr/bin, but I've left it as is, since it's hard-coded in `overlaycheck`. I'll add a patch in the packaging to resolve that without affecting people who use `overlaycheck` directly from the git checkout.

Reply:
The script is essentially architecture-independent data.

Reply:
In the literal sense that it is just a text file in the end, or in the FHS sense that you don't expect users to run it directly?

When it comes to FHS and cmake install paths, it's a user executable, unless it's a helper script that gets run by something else.

Reply:
In the literal sense.